.github/workflows/unit_tests_with_latest_deps.yml

```diff
@@ -37,36 +42,65 @@ jobs:
-        name: Install woodwork - spark requirements
+      - name: Install Dask and Spark
         run: |
           sudo apt update
           sudo apt install -y openjdk-11-jre-headless
           JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
           python -m pip install unpacked_sdist/[spark]
-      - if: ${{ matrix.libraries == 'dask' }}
-        name: Install woodwork - dask requirements
-        run: |
           python -m pip install unpacked_sdist/[dask]
-      - if: ${{ matrix.python_version != 3.8 }}
-        name: Run unit tests (no code coverage)
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'Testing to Disk with LatLong' }}
+        name: Run testing to Disk with LatLong Unit Tests (no code coverage)
         run: |
           cd unpacked_sdist
-          pytest woodwork/ -n 2
-        env:
-          PYARROW_IGNORE_TIMEZONE: 1
-          ALTERYX_OPEN_SRC_UPDATE_CHECKER: False
-      - if: ${{ matrix.python_version == 3.8 }}
-        name: Run unit tests with code coverage
+          pytest woodwork/tests/accessor/test_serialization.py::test_to_disk_with_latlong -n 2 --durations 0
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'All other Serialization' }}
+        name: Run all other Serialization Unit Tests (no code coverage)
+        run: |
+          cd unpacked_sdist
+          pytest woodwork/tests/accessor/test_serialization.py --ignore=woodwork/tests/accessor/test_serialization.py::test_to_disk_with_latlong -n 2 --durations 0
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'Testing Table Accessor' }}
+        name: Run Table Accessor Unit Tests (no code coverage)
+        run: |
+          cd unpacked_sdist
+          pytest woodwork/tests/accessor/test_table_accessor.py -n 2 --durations 0
+      - if: ${{ matrix.python_version != 3.8 && matrix.directories == 'All Other Tests' }}
+        name: Run all other Unit Tests (no code coverage)
+        run: |
+          cd unpacked_sdist
+          pytest woodwork/ -n 2 --ignore=woodwork/tests/accessor/test_serialization.py --ignore=woodwork/tests/accessor/test_table_accessor.py --durations 0
+      - if: ${{ matrix.python_version == 3.8 && matrix.directories == 'Testing to Disk with LatLong' }}
+        name: Run Serialization Unit Tests with code coverage
+        run: |
+          python -m pip install "$(cat setup.cfg | grep codecov)"
+          coverage erase
+          cd unpacked_sdist/
+          coverage erase
+          pytest woodwork/tests/accessor/test_serialization.py::test_to_disk_with_latlong -n 2 --durations 0 --cov=woodwork --cov-config=../.coveragerc
+      - if: ${{ matrix.python_version == 3.8 && matrix.directories == 'All other Serialization' }}
+        name: Run Serialization Unit Tests with code coverage
+        run: |
+          python -m pip install "$(cat setup.cfg | grep codecov)"
+          coverage erase
+          cd unpacked_sdist/
```

Comment:
Similarly for these 4 calls (before `pytest woodwork...`, can we run these before the `if` statements or outside of that logic? Would allow us to clean this up a little bit and make the file a bit more readable.

Reply:
I think there's a limit to what we can put in certain steps because each step is n individual instance.

Reply:
Sounds good.